src/test/debuginfo/no_mangle-info.rs

```diff
@@ -0,0 +1,37 @@
+// compile-flags:-g
+
+// === GDB TESTS ===================================================================================
+// gdb-command:run
+// gdb-command:p TEST
+// gdb-check:$1 = 3735928559
+// gdb-command:p no_mangle_info::namespace::OTHER_TEST
+// gdb-check:$2 = 42
+
+// === LLDB TESTS ==================================================================================
+// lldb-command:run
+// lldb-command:p TEST
+// lldb-check: (unsigned long) $0 = 3735928559
+// lldb-command:p OTHER_TEST
```

Comment:
So it turns out LLDB functions differently and expects the unqualified name for this symbol... I'm not sure how symbol resolution works in LLDB, but the debug info generated by rustc seems to be correct.

Reply:
Looking at [this reference](https://lldb.llvm.org/use/map.html#examining-variables), it seems that the corresponding lldb command should be `target variable OTHER_TEST` (or, perhaps the fully qualified name). It seems strange that it would for one of these but not the other but perhaps we're just using the wrong command?